lib/plugin_gem.rb

```diff
@@ -16,7 +16,10 @@ def self.load(path, name, version, opts = nil)
       command  = "gem install #{name} -v #{version} -i #{gems_path} --no-document --ignore-dependencies --no-user-install"
       command += " --source #{opts[:source]}" if opts[:source]
       puts command
-      puts `#{command}`
+
+      Bundler.with_unbundled_env do
```

Comment:
Not testing this because we do not want to be installing gems in our test suite.